packages/app/package.json

```diff
@@ -51,37 +51,37 @@
     "ember-cli-inject-live-reload": "^1.8.2",
     "ember-cli-less": "^1.5.5",
     "ember-cli-mirage": "~0.4.9",
-    "ember-cli-moment-shim": "3.5.0",
+    "ember-cli-moment-shim": "^3.7.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-string-helpers": "^1.6.0",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
-    "ember-cli-update": "^0.29.2",
+    "ember-cli-update": "^0.34.10",
     "ember-collection": "1.0.0-alpha.9",
     "ember-composable-helpers": "^2.1.0",
     "ember-cp-validations": "^3.5.0",
     "ember-debounced-properties": "0.0.5",
     "ember-export-application-global": "^2.0.0",
     "ember-font-awesome": "^3.0.5",
     "ember-get-config": "0.2.4",
-    "ember-gridstack": "1.2.0",
+    "ember-gridstack": "1.3.0",
     "ember-load-initializers": "^1.1.0",
-    "ember-lodash": "^4.19.4",
+    "ember-lodash": "^4.19.5",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-modal-dialog": "^2.4.1",
     "ember-moment": "7.5.0",
     "ember-pluralize": "0.2.0",
     "ember-power-select": "^2.2.3",
-    "ember-promise-helpers": "^1.0.3",
+    "ember-promise-helpers": "^1.0.6",
     "ember-qunit": "^4.3.0",
     "ember-radio-button": "^1.2.1",
     "ember-resize": "0.3.4",
     "ember-resolver": "^5.0.1",
     "ember-route-action-helper": "^2.0.6",
     "ember-sortable": "1.12.3",
-    "ember-tether": "^1.0.0",
```

Comment:
do we not need this?

Reply:
It was moved out of dev-dependencies.
https://github.com/sir-dunxalot/ember-tooltips/releases/tag/2.9.0